cw_bitcoin/lib/electrum_wallet.dart

```diff
@@ -1362,26 +1363,11 @@ abstract class ElectrumWalletBase
     }
   }
 
-  Future<bool> canReplaceByFee(String hash) async {
-    final verboseTransaction = await electrumClient.getTransactionVerbose(hash: hash);
-
-    final String? transactionHex;
-    int confirmations = 0;
-
-    if (verboseTransaction.isEmpty) {
-      transactionHex = await electrumClient.getTransactionHex(hash: hash);
-    } else {
-      confirmations = verboseTransaction['confirmations'] as int? ?? 0;
-      transactionHex = verboseTransaction['hex'] as String?;
-    }
-
-    if (confirmations > 0) return false;
-
-    if (transactionHex == null || transactionHex.isEmpty) {
-      return false;
-    }
-
-    return BtcTransaction.fromRaw(transactionHex).canReplaceByFee;
+  Future<bool> canReplaceByFee(ElectrumTransactionInfo tx) async {
+    final bundle = await getTransactionExpanded(hash: tx.txHash);
+    _updateInputsAndOutputs(tx, bundle);
+    if (bundle.confirmations > 0) return false;
+    return bundle.originalTransaction.canReplaceByFee;
```

Comment:
how would this handle if it's called on a transaction that's already replaced?

Reply:
Added a try-catch block to handle this exception.